application/src/main/java/org/opentripplanner/apis/gtfs/datafetchers/stepImpl.java

```diff
@@ -57,7 +61,24 @@ public DataFetcher<String> exit() {
 
   @Override
   public DataFetcher<Object> feature() {
-    return environment -> getSource(environment).entrance().orElse(null);
+    return environment -> {
+      WalkStep walkStep = getSource(environment);
+      if (walkStep.entrance().isPresent()) {
+        return walkStep.entrance().get();
+      } else if (walkStep.getEdges().size() == 1) {
```

Comment:
If a step has a single edge, does that always mean it's an escalator or elevator? Seems quite dubious.

Reply:
this has been reverted back to the old implementation with a field in WalkStep